include/votca/tools/histogramnew.h

```diff
@@ -169,19 +162,14 @@ class HistogramNew {
   void setPeriodic(bool periodic) { _periodic = periodic; }
```

Comment:
I would recommend throwing an error if the periodicity is changed at some point after data is put in the table. So that some one is not accessing data that they think is periodic when it is no longer.

Reply:
or we reset the data and initialize again.

Reply:
Reset when a setting is changed?

Reply:
a) throwing an error, requires knowledge if processing has happened already
b) requires that we initialize before setting periodicity
c) the current version works with the current code (I hope)

Reply:
It will, probably, but I think it might be inviting future problems, and that would be a hard bug to find. I think some kind of check should be in place. 

If you have to manually reinitialize after changing a setting that is problematic, unless the code fails catastropically you might very well forget to reinitialize.

Reply:
Can we make the periodic bool part of the initialisation, so it would all be together, maybe as a default=false, I would also rename initliasation to SetupBins

Reply:
Sounds good to me.

Reply:
Unfortunately, Changing the Interface of histogram is problematic as it involves csg and xtp and tools. Can we merge it as it is and then I'll change it later? @junghans @JoshuaSBrown